docs/api/portable-stories/portable-stories-jest.mdx

```diff
@@ -159,26 +159,18 @@ draft: true
   ## setProjectAnnotations
 
   This API should be called once, before the tests run, typically in a [setup file](https://jestjs.io/docs/configuration#setupfiles-array). This will make sure that whenever `composeStories` or `composeStory` are called, the project annotations are taken into account as well.
+  
+  These are the configurations needed in the setup file:
+  - preview annotations: those defined in `.storybook/preview.ts`
+  - addon annotations (optional): those exported by addons
+  - beforeAll: code that runs before all tests
```

Comment:
Would be great to point this to the `beforeAll` documentation, which we unfortunately do not have yet. I'm adding this comment here as a reminder for the future

Reply:
Docs for `beforeAll` were added in https://github.com/storybookjs/storybook/pull/28444, so I added a link. 🎉